### ISSUE TITLE
Undefined SASS variable - $helvetica

### ISSUE THREAD

**Issue by herrjosua** [18F/web-design-standards#2229]
## Description

I'm trying to setup the Web Design Standards on a project I'm working on, but I'm encountering an issue when I try to run my local instance using Angular-CLI. It is currently telling me that `$helvetica` variable is undefined. I've successfully overridden the `fonts` and `img` directories based on what my team has developed for the projects structure.

I have tried doing a search for this variable with my IDE (Webstorm), but I have not yet figure out how it works within the IDE.  I see it being used in the `_variables.scss`, but not where it being defined. Any help will be greatly appreciated.

**Comment by maya** [comment]
Hi @herrjosua, thanks for reaching out. I believe that is coming from the Bourbon library which is included in the project. See: http://bourbon.io/docs/#font-stacks Is that library being included in your implementation?

**Comment by herrjosua** [comment]
@maya No, I will have to look into adding that into the project. Thanks

**Comment by maya** [comment]
Sounds good, closing for now, but feel free to re-open if that doesn't solve your problem. Good luck!

**Comment by herrjosua** [comment]
@maya I'm still encountering the issue. I installed bourbon per the instructions, but I still get the error. I also tried importing the bourbon file before importing the the USWDS framework and my custom sass files. When I add both the import for the USWDS framework and the bourdon I get errors that it can not find the images and fonts directories. The second error is when I remove the bourdon import I get the `$helvetica` variable is undefined.
